src/fidesops/service/connectors/query_config.py

```diff
@@ -497,6 +520,42 @@ def get_formatted_query_string(
         return f'SELECT {field_list} FROM "{self.node.node.collection.name}" WHERE {" OR ".join(clauses)}'
 
 
+class BigQueryQueryConfig(QueryStringWithoutTuplesOverrideQueryConfig):
+    """
+    Generates SQL valid for BigQuery
+    """
+
+    def generate_update(
+        self, row: Row, policy: Policy, request: PrivacyRequest, client: Engine
+    ) -> Optional[Update]:
+        """
+        Using TextClause to insert 'None' values into BigQuery throws an exception, so we use update clause instead
+        """
```

Comment:
Can you add to this description that it returns a SQLAlchemy Update object, that the update itself is not occurring here?  mostly for clarification because sqlalchemy's syntax itself can be confusing - table.update().where(*pk_clauses).values(**update_value_map)

Reply:
agreed, good idea!